code/modules/mob/living/silicon/robot/robot_modules.dm

```diff
@@ -194,7 +195,7 @@ var/global/list/robot_modules = list(
 				"Heavy" = "heavyMed",
 				"Needles" = "medicalrobot",
 				"Standard" = "surgeon",
-				"Advanced Droid - Medical" = "droid-surgery",
+				"Advanced Droid - Medical" = "droid-medical",

```

Comment:
I think agent whatever caused this. 

I didnt see any others, hopefully it was an isolated mistake